src/ripple/app/main/GRPCServer.cpp

```diff
@@ -450,13 +450,16 @@ GRPCServer::run()
     }
 }
 
-GRPCServer::~GRPCServer()
```

Comment:
Since we're now tracking `running_` correctly, I suggest we give `~GRPCServer()` a non-trivial body:
```
GRPCServer::~GRPCServer()
{
    assert (!running_);
}
```